Tmain/json-output-format.d/stdout-expected.txt

```diff
@@ -16,7 +16,7 @@
 {"_type": "tag", "name": "main", "path": "input.go", "pattern": "/^package main$/", "language": "Go", "line": 1, "kind": "package"}
 # json --languages=+man --fields=* --extras=*-{subword}
 {"_type": "ptag", "name": "JSON_OUTPUT_VERSION", "path": "0.0", "pattern": "in development"}
-{"_type": "ptag", "name": "TAG_FILE_FORMAT", "path": "2", "pattern": "extended format; --format=1 will not append ;\" to lines"}
+{"_type": "ptag", "name": "TAG_FILE_FORMAT", "path": "3", "pattern": "extended format with meta character escaping"}
 {"_type": "ptag", "name": "TAG_FILE_SORTED", "path": "1", "pattern": "0=unsorted, 1=sorted, 2=foldcase"}
```

Comment:
Nothing tod owith output format but I wonder soriting json output makes sense or not.